modules/data-protection.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * service_description/rhacs-cloud-service-service-description.adoc
+:_content-type: CONCEPT
+[id="data-protection_{context}"]
+= Data protection
```

Comment:
I think you need a short description here: https://redhat-documentation.github.io/supplementary-style-guide/#shortdesc